### PR TITLE
#10 handle save for answer_edit view

### LINKED ISSUE #10: handle save for answer edit view
* save action should write to database the following object
```
{
  'key': 'username'
  'answers': [ 'answer_1', 'answer_2' ]
}
```
* part of issue #4

### PR DESCRIPTION
**Pull request by Suppi14**
- Persist answer to DB: `ccm.answer_edit.js`
- resolve #10

### CODE REVIEW

File: answer_edit/ccm.answer_edit.js [hunk 1]
```diff
@@ -105,17 +105,18 @@
         // get page fragments
         const contentElem = self.element.querySelector( '#content' );
         const saveElem = self.element.querySelector( '#save' );
-
+        // const qaId = null;
```
Comment (minhnh) [inline answer_edit/ccm.answer_edit.js]: please remove commented out code

File: answer_edit/ccm.answer_edit.js [hunk 2]
```diff
@@ -105,17 +105,18 @@
...
         // load questions from store
         await self.data.store.get(self.constants.key_questions).then( (questions) => {
           questions && questions.entries && questions.entries.forEach( (entry, i) => {
             const qaId = self.constants.qa_prefix + i;
             qaData[qaId] = {};
             qaData[qaId]['question'] = entry.text;
+            console.log(qaData[qaId]['question']);
```
Comment (minhnh) [inline answer_edit/ccm.answer_edit.js]: please remove debug code

File: answer_edit/ccm.answer_edit.js [hunk 3]
```diff
@@ -105,17 +105,18 @@
...
           } );
         });
 
-        renderQAPairs();
+         renderQAPairs();
```
Comment (minhnh) [inline answer_edit/ccm.answer_edit.js]: remove this unnecessary change

File: answer_edit/ccm.answer_edit.js [hunk 4]
```diff
@@ -134,16 +154,16 @@
 
             qaDiv.innerHTML = qaDiv.innerHTML.replace(/\$qa_id\$/g, qaId);
             qaDiv.innerHTML = qaDiv.innerHTML.replace(/\$qa_question\$/g, qaData[qaId].question);
-            const answer = qaData[qaId].answer ? qaData[qaId].answer : '';
+            const answer = qaData[qaId].text ? qaData[qaId].text : '';
             qaDiv.innerHTML = qaDiv.innerHTML.replace(/\$qa_answer\$/g, answer);
             contentElem.appendChild(qaDiv);
           } );
-        }
+         }
```
Comment (minhnh) [inline answer_edit/ccm.answer_edit.js]: remove unnecessary change

**Review by minhnh — CHANGES_REQUESTED** [review]
Only minor changes. If you can show me how it works on your computer, we can merge this.

**Comment by Suppi14** [comment]
Yes. I can show the working

**Review by minhnh — APPROVED** [review]

**Pull request merged**